src/components/Global/BaseSnackbar.vue

```diff
@@ -7,7 +7,7 @@
   <v-snackbar
     :modelValue="modelValue"
     @update:modelValue="$emit('update:modelValue', $event)"
-    :color="color"
+    :color="color || 'error'"
```

Comment:
This is the right idea and should work, but the more traditional and semantic way to set a default value for the prop is to configure the prop definition itself. So we could keep this line as is
```suggestion
    :color="color"
```

Then update the props definition on line 34, so that instead of using the primitive array-of-strings syntax , we could use the object syntax which lets us specify a default value or whether a prop is required. Check the [vue docs](https://vuejs.org/guide/components/props.html#prop-validation) on prop validation. 

This way we could specify that `modelValue`, `message`, and `data-cy` are `required` props, but `color` instead has `default: 'error'`